contracts/truefi2/Liquidator2.sol

```diff
@@ -145,6 +145,10 @@ contract Liquidator2 is UpgradeableClaimable {
      */
     function liquidate(IDebtToken[] calldata loans) external {
         require(msg.sender == SAFU, "Liquidator: Only SAFU contract can liquidate a loan");
+        require(
+            areLoansOfSingleBorrower(loans),
```

Comment:
How about this naming instead?
```suggestion
            allLoansHaveSameBorrower(loans),
```
If you like this, please update the error message text as well.